packages/twenty-zapier/src/utils.ts

```diff
@@ -0,0 +1,52 @@
+import { Bundle, HttpRequestOptions, ZObject } from 'zapier-platform-core';
+export const getBundle = (inputData?: object) => {
+  return {
+    authData: { apiKey: String(process.env.API_KEY) },
+    inputData,
+  };
+};
+export const handleQueryParams = (inputData: { [x: string]: any }): string => {
```

Comment:
usually, we prefer to have one function per file: /utils/handleQueryParams.ts, /utils/requestDb per file. It helps scaling the project smoothly